prime-router/settings/ca-scc-phd.yml

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
Can't tell if this is correct because I can't tell if this is destined for -local, -test, or -prod.